Tests/Unit/Search/Adapter/ZendLuceneAdapterTest.php

```diff
@@ -197,11 +203,32 @@ public function testIndexWithNullValues()
         $this->assertNotContains('field2', $luceneDocument->getFieldNames());
     }
 
+    public function testInitialize()
+    {
+        $adapter = $this->createAdapter($this->dataPath);
+
+        $this->filesystem->exists($this->dataPath)->willReturn(false);
+        $this->filesystem->mkdir($this->dataPath)->shouldBeCalled();
+
+        $adapter->initialize();
+    }
+
+    public function testInitializeWithExistingFolder()
```

Comment:
Would prefer if you added comments to these tests as with the others in the form of `It should ...`